decidim/decidim
# Fix not showing extra attributes in assembly when it's empty

#### :tophat: What? Why?

There's a bug in the Assemblies' main data content block (enabled by default in the seeds), where if some of the extra attributes that are shown in the description are empty, the title is still shown. 

This PR fixes it. It also introduces a spec for this CB that was missing.  

#### Testing

1. Sign in as admin
2. Edit an assembly and remove the "Purpose of action" in the edit form
3. Go to the landing page with the "Main data" content block, and see that the "Purpose of action" isn't shown anymore

### :camera: Screenshots

#### Before
![Screenshot of the assembly landing page with the bug](https://github.com/decidim/decidim/assets/717367/4ea968c4-2173-4869-b06f-0bce14c48e63)

#### After
![Screenshot of the assembly landing page with the bug fixed](https://github.com/decidim/decidim/assets/717367/3368268c-4ba6-4285-a2e0-d15afaee81d8)

:hearts: Thank you!
